lib/matplotlib/pyplot.py

```diff
@@ -41,8 +41,6 @@
 import importlib
 import inspect
 import logging
-from numbers import Number
-import re
```

Comment:
Similar here: if someone imports `Number` or `re` from `pyplot`, it is, well, not correct to start with.